lib/sbom/cyclonedx/json.ex

```diff
@@ -0,0 +1,46 @@
+defmodule SBoM.CycloneDX.Json do
+  @moduledoc false
+
+  alias alias SBoM.License
+
+  def bom(components, opts) do
+    %{
+      bomFormat: "CycloneDX",
+      specVersion: opts[:schema] || "1.2",
+      serialNumber: opts[:serial] || SBoM.CycloneDX.uuid(),
+      version: "1",
+      metadata: _metadata(),
+      components: components(components)
+    }
+    |> Jason.encode_to_iodata!(pretty: true)
+  end
+
+  defp _metadata do
```

Comment:
For consistency we can remove the uderscore `_`

Reply:
@sigu I have worked on it